### ISSUE TITLE
Large Provinces Breaking Down: Soil and Weather Creation

### ISSUE THREAD

**Issue by smkuhlani** [AgWise-EiA/potentialyield#6]
There are challenges in creating soil and weather files for large areas. This had been solved this by splitting the soil and weather file creation by Zones/provinces. There is now a bottle neck again, when if we have zones/provinces with more than 3000/4000 points the generation of soil and weather files breaks. We would like to have ideas on how to solve the challenge. @lpmorenoc  , @asila , @meklitC

**Comment by lpmorenoc** [comment]
@smkuhlani does it happen even if you run the R script from the command line?

**Comment by smkuhlani** [comment]
Yes, it happens if you run from the terminal. I have just tested it now and it crashed. 





![image](https://github.com/AgWISE-EiA/potentialyield/assets/34687653/d4152d1d-47ba-4524-8f29-303b61099ed2)






Though i have to admit, such a large process has worked before, a few weeks ago.

**Comment by smkuhlani** [comment]
/agwise-potentialyield/dataops/potentialyield/Script/useCases/UseCase_Zambia_Solidaridad/Maize/DSSAT/get_CM_geo_Zambia_Solidaridad_Maize.R

@lpmorenoc , this one!
